js/telemetry/trackingEvents.ts

```diff
@@ -0,0 +1,55 @@
+import { SideBarSelection } from "../components/ladderPage/sidebar";
+import { OCSTrip } from "../models/ocs";
+import { estimatedArrivalOfVehicle } from "../models/vehicle";
+import { FullStory } from "@fullstory/browser";
+
+export enum FullStoryEventName {
+  SideBarOpened = "Preston test event",
+  // TODO: Change to
+  // SideBarOpened = "Orbit: Ladder Side Bar Opened"
+}
+
```

Comment:
I will change this to the real event name before merge. Please let me know if anyone has feedback on the actual name to be used. I am considering prefixing it with "Orbit:" as shown, since events in Fullstory are shared across the whole org and there doesn't seem to be a good way to organize them by application in certain contexts.

Reply:
Seems good to me 👍 `[Application]:` is also what we do in Splunk for the same reason.